pkgs/tools/inputmethods/ibus-engines/ibus-mozc/default.nix

```diff
@@ -21,32 +22,22 @@ in clangStdenv.mkDerivation rec {
     maintainers  = with maintainers; [ gebner ericsagnes ];
   };
 
-  nativeBuildInputs = [ which ninja python gyp pkgconfig qt5.wrapQtAppsHook ];
+  nativeBuildInputs = [ which ninja python python37Packages.six gyp pkgconfig qt5.wrapQtAppsHook ];
   buildInputs = [ protobuf ibus gtk2 zinnia qt5.qtbase libxcb ];
 
   src = fetchFromGitHub {
     owner  = "google";
     repo   = "mozc";
-    rev    = "afb03ddfe72dde4cf2409863a3bfea160f7a66d8";
-    sha256 = "0w2dy2j9x5nc7x3g95j17r3m60vbfyn5j617h7js9xryv33yzpgx";
+    rev    = "91cc1e19ef34aeb12888b697fefa52907f1a834d";
+    sha256 = "1fyy9g1pnaq6s5nkf56aqmp5mgyibbmp1ylc64fqc6g1plg90zk2";
+    fetchSubmodules = true;
   };
 
-  patches = [
-    # https://github.com/google/mozc/pull/444 - fix for gcc8 STL
-    (fetchpatch {
-      url = "https://github.com/google/mozc/commit/82d38f929882a9c62289b179c6fe41efed249987.patch";
-      sha256 = "07cja1b7qfsd3i76nscf1zwiav74h7d6h2g9g2w4bs3h1mc9jwla";
-    })
-    # Support dates after 2019
-    (fetchpatch {
-      url = "https://salsa.debian.org/debian/mozc/-/raw/master/debian/patches/add_support_new_japanese_era.patch";
-      sha256 = "1dsiiglrmm8i8shn2hv0j2b8pv6miysjrimj4569h606j4lwmcw2";
-    })
-  ];
-
   postUnpack = ''
-    rmdir $sourceRoot/src/third_party/japanese_usage_dictionary/
+    rm -rf $sourceRoot/src/third_party/japanese_usage_dictionary/
```

Comment:
This is fine like this. Adding -f is not great because we won't notice if the directory layout changes.

Reply:
We now fetch submodules that are required in order to build mozc, as such the directory is not empty anymore and rmdir won't do